src/App.jsx

```diff
@@ -14,33 +15,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goodsList, setGoodsList] = useState([...goodsFromServer]);
+  const [sortType, setSortType] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const applySorting = (type, reversed = false) => {
```

Comment:
where i return?
all function should have return

you should have smtn like:

const visibleGoods =  preparedGoods () ... logic with return inside function




```suggestion
  const preparedGoods = (type, reversed = false) => {
```